CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing
+Hi! 
+
+Thanks for considering participation in WarsawLO project!
+We believe that tools like ours should be crafted by the community! 
+
+We hope that this guide will help you find your own way of participation.
+If you still have questions after reading this, feel free to reach us.
```

Comment:
If this document does not answer your questions, feel free to reach us at <rzeczy>